cassandra-diagnostics-core/src/main/java/io/smartcat/cassandra/diagnostics/module/requestrate/RequestRateModule.java

```diff
@@ -94,21 +123,34 @@ private double convertRate(double rate) {
     private class RequestRateTask extends TimerTask {
         @Override
         public void run() {
-            for (StatementType statementType : requestRates.keySet()) {
-                double requestRate = convertRate(requestRates.get(statementType).sumThenReset());
-
-                logger.debug("{} request rate: {}/{}", statementType, requestRate, timeunit.name());
-
-                report(createMeasurement(service, statementType, requestRate));
+            for (String statementConsistency : requestRates.keySet()) {
+                double requestRate = convertRate(requestRates.get(statementConsistency).sumThenReset());
+
+                if (ALL_REQUESTS_TO_REPORT.equals(statementConsistency)) {
```

Comment:
Same as above, no need to distinguish with ALL_REQUESTS_TO_REPORT if its simply being matched.